### ISSUE TITLE
Notabene 9 cannot open or write a new file

### ISSUE THREAD

**Issue by PBreeze** [otya128/winevdm#592]
Try the latest version first.
Go [here](https://ci.appveyor.com/project/otya128/winevdm) and click `Environment: THIS_BUILD_IS_RECOMMENDED__VCXPROJ_BUILD=1`-> `Artifacts 1`-> `zip` to download the latest version.

Notabene 9 will launch correctly with 1611 but if I try to open a file I immediately get an error such as the one below:

![nb error 1](https://user-images.githubusercontent.com/59335560/72446148-e68a1b00-37aa-11ea-82a6-362ba9127c9a.jpg)

If I try using the menu bar/file/new to start a new file I get an OTVDM error:

![nb error 2](https://user-images.githubusercontent.com/59335560/72446247-1afdd700-37ab-11ea-8849-6734bdf49d63.jpg)

I have tried a trace but when I launch the trace with notabene.exe, the trace.txt file just keeps     growing.  Last time it reached 126MB and was still climbing when an error forced closure of the program.

**Describe the bug**

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - OS: [e.g. Windows 10 1903]
 - Version [e.g. version: 1523]

**Additional context**

It is very helpful to provide a trace file if possible.
Please note that this file contains detailed information about the program.
**Procedure**
1. Run the command prompt.
2. Set the environment variable:
```cmd
set WINEDEBUG=+all,-snoop,-ldt,-fixup,-module,-global,-local,-disasm,-syslevel,-thunk
```
3.
```cmd
*old-executable-file* 2> trace.txt
```
4. Upload trace.txt here.

**Comment by PBreeze** [comment]
Should this be merged with #585?

**Comment by cracyc** [comment]
With the current top-of-tree the program is functional although not quite stable with occasional crashes.

**Comment by PBreeze** [comment]
I have just downloaded and tried 1627.  You are right, Notabene 9 is more stable now.  I can write text in the open document without crashing.  When I first tried with 1627 I could open some large files with fottnotes and table that were written using the program by accessing the menu bar File/Open dialog box.  Scrolling through a document was slow but it worked.  However various SEGV errors did start to appear quite quickly.  Since that first time, I have not been able to open a file using the menu.  Instead, when I try to use the File/Open dialog and click on a file to open it, the program hangs.  On the other hand, the File/Open dialog box does now have a file system that I can read whereas before it was full of strange characters.  I have had more success when opening and closing files using the command line.  However reverting to the File/Close option to close an open file generates another SEGV error.

My Event viewer contains some error reports for Explorer.exe and otvdmw.exe that were generated by running notabene 9.  Are these of any value?  If so I can take copies and upload them.

This does look much more promising now, so thanks for the hard work so far.

**Comment by PBreeze** [comment]
I have been exploring the current state of Notabene 9 a little more and its seems to me that most of the errors are generated from the menus and dialog boxes.  I can call, edit and save files from the command line that is available in the program without generating errors.

**Comment by PBreeze** [comment]
(This is with otvdm-master-1638) I am going to try and work through errors in Notabene 9 one at a time.  The most important error I have encountered at the moment is that if I try to use the Menu/File/Open dialog the program freezes.  This happens either when trying to nativate using the file system tree or when clicking on a file in the file list -- or I have just found, if I try to exit the dialog without doing anything.  It is curious that the very first time I tried this after a clean install of Notabene 9 I could open files, but since then, for over a week now, it has been impossible.  There is no error message to report, just the program not responding.  I have to kill it to recover.  Would it be possible to try and fix this as a priority for this program?  Thanks.

**Comment by cracyc** [comment]
https://github.com/otya128/winevdm/pull/658 fixes the new file problem, I can't repo the open file problem but there are somewhat random problems which are hard to nail down

**Comment by PBreeze** [comment]
Yes, it does seem a little more stable with this fix, but I too still get some random errors, even with file opening.

**Comment by cracyc** [comment]
If you can reliably reproduce any problems let us know.
